.github/workflows/pr-test.yml

```diff
@@ -85,3 +85,43 @@ jobs:
       - name: Run integration tests
         run: |
           pipenv run pytest integration_tests -k 'not api_key'
+
+  performance-tests:
+    strategy:
+      fail-fast: true
+      matrix:
+        python: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
@tronxd Won't the runtime be a little different in each python version? What does running it on all python runtimes give us?

I'd expect we would run only on 1 runtime, doesn't matter which one IMO